resource-managers/yarn/pom.xml

```diff
@@ -126,7 +126,7 @@
     </dependency>
 
      <!--
-    See SPARK-3710. hadoop-yarn-server-tests in Hadoop 2.2 fails to pull some needed
```

Comment:
I wonder if we can just remove everything after the `hadoop-yarn-server-tests` of if they're still needed because of the way we do exclusions elsewhere?

The original issue this comment was addressing does not exist after Hadoop 2.2, so the wording here is, at best, a little stale now.